tests/tts_tests/test_tacotron2_train_fsspec_path.py

```diff
@@ -0,0 +1,55 @@
+import glob
```

Comment:
How is it different than a regular training test except for the `file` paths ?

Reply:
It's exactly the same :) I could also add "file://" paths to the original one (without the `fsspec_path` suffix), do you have a preference? This will have a test execution time penalty but it might perhaps be easier to tell if `fsspec` is the culprit.